sceptre/nlpsandbox/config/common/nlpsandboxvpc.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: remote/vpc-mini.yaml
+stack_name: nlpsandboxvpc
+stack_tags:
+  Department: "CompOnc"
+  Project: "Nlp Sandbox"
+  OwnerEmail: "thomas.yu@sagebase.org"
+parameters:
+  VpcSubnetPrefix: "10.255.20"
```

Comment:
@zaro0508 , is this something that is supplied to us?

Reply:
Yes, IT Ops would give you a CIDR block to use. If a minivpc schema (256 ipv4 addrs divided into small vpcs of 32 addresses) works for this project, go ahead and use "10.255.21" as the prefix and we'll mark in on the allocation list.

Reply:
refer to wiki and find a value that's not used, https://sagebionetworks.jira.com/wiki/spaces/IT/pages/352976898/Sage+VPN

I see `10.255.31` isn't listed so let's go with that?  also if you use that value please update that info on the wiki.